inspector/src/components/graph/canvasGraphService.ts

```diff
@@ -484,7 +571,7 @@ export class CanvasGraphService {
     private _drawPlayheadRegion(drawableArea: IGraphDrawableArea, scaleFactor: number) {
         const { _ctx: ctx } = this;
        
-        if (!ctx) {
+        if (!ctx || scaleFactor >= 0.95) {

```

Comment:
sounds like a magic number to me ?

Reply:
Good catch! I forgot that I do not carry a license in witchcraft and wizardry from Hogwarts. So, I have turned this magical number (and another) into a constant and added comment explaining what it is for in my latest commit. 😄